css-transitions-2/Overview.bs

```diff
@@ -167,6 +167,78 @@ Each time a new transition is generated, the current value of the (already
 incremented) <a>current transition generation</a> is stored as the
 transition's <dfn>transition generation</dfn>.
 
+## Defining [=before-change style=]: the ''@starting-style'' rule
+
+Transitions can only start during a [=style change event=] for elements which
+has a defined [=before-change style=] established by the previous
+[=style change event=], which means that newly inserted elements or elements
+that change from not [=being rendered=] to being rendered do not start
+transitions. Still, doing transitions starting from the initial rendering is a
+valid use case.
+
+The <dfn at-rule id="at-ruledef-starting-style">@starting-style</dfn> rule is a
+conditional group rule with the implicit condition that it matches if the
+previous [=style change event=] did not establish a [=before-change style=] for
+the element whose styles are being computed. This is used to establish a
+[=before-change style=] for the current [=style change event=].
```

Comment:
I think it's probably worth adding an informative note here that this means that a style computation for an element that does use `@starting-style` rules usually inherits from (maybe not directly, but somewhere in its ancestor chain) from an element that doesn't use `@starting-style` rules because it had a before-change style already.  (I assume this is how we want this to work?)

Reply:
Yes, and also that the before-change style that is set from @starting-style actually inherits from the after-change style of the parent, even if the parent has @starting-style applied. Does that make sense?

Reply:
I don't think that's correct (see comment below).